rpcs3/Emu/RSX/GL/GLPresent.cpp

```diff
@@ -332,7 +332,7 @@ void GLGSRender::flip(const rsx::display_flip_info_t& info)
 		{
 			const f32 gamma = avconfig.gamma;
 			const bool limited_range = !use_full_rgb_range_output;
-			const auto filter = m_output_scaling == output_scaling_mode::nearest ? gl::filter::nearest : gl::filter::linear;
+			const gl::filter filter = m_output_scaling == output_scaling_mode::nearest ? gl::filter::nearest : gl::filter::linear;
```

Comment:
This really isn't necessary. It can't be anything other than a filter. Is there a confused compiler out there?